go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/fsnotify/fsnotify
 go 1.13
 
 require golang.org/x/sys v0.0.0-20191005200804-aed5e4c7ecf9
+
+replace golang.org/x/sys => github.com/nshalman/sys v0.0.0-20210702004434-76c4cd66abf6
```

Comment:
This needs to be removed by bumping the dependency above after the Event Ports implementation lands.

Reply:
It landed today!